website/docs/r/nsxv_firewall_rule.markdown

```diff
@@ -185,6 +185,7 @@ accepted as a parameter.
 * `gateway_interfaces` - (Optional) A set of with either three keywords `vse` (UI names it as `any`), `internal`, `external` or an org network name. It automatically looks up vNic in the backend.
 * `virtual_machine_ids` - (Optional) A set of `.id` fields of `vcd_vapp_vm` resources.
 * `org_networks` - (Optional) A set of org network names.
+* `ipsets` - (Optional) A set of IP set names.
```

Comment:
Maybe some elaborated explanation needed from where these ip set names comes/configured

Reply:
Added - double check if it makes sense.